docker/Dockerfile.ubuntu2204

```
@@ -0,0 +1,74 @@
+#!/usr/bin/docker
+# ██████  ███████ ██████   ██████  ███████ ███████ ███    ██ ███████ ███████
+# ██   ██ ██      ██   ██ ██    ██ ██      ██      ████   ██ ██      ██
+# ██████  █████   ██████  ██    ██ ███████ █████   ██ ██  ██ ███████ █████
+# ██   ██ ██      ██      ██    ██      ██ ██      ██  ██ ██      ██ ██
+# ██   ██ ███████ ██       ██████  ███████ ███████ ██   ████ ███████ ███████
+#
+# Title:            Ubuntu LTS 22.04 Builds
+# Author:           George Tay
+# Version:          0.1
+# License:          MIT (see main repository for Licensing information
+#
+# Dockerfile format inspired by https://github.com/sickcodes/Docker-OSX/blob/master/Dockerfile.naked
+#
+# These are some of the resources which I heavily referenced to create this Dockerfile:
+# - Passing Arguments into Docker Containers
+#     - https://stackoverflow.com/questions/40873165/use-docker-run-command-to-pass-arguments-to-cmd-in-dockerfile
+# - Installation of Java on Ubuntu
+#     - https://stackoverflow.com/questions/49914574/install-jdk-8-update-172-in-dockerfile-with-ubuntu-image
+# - Installation of NodeJS via Node Version Manager (NVM)
+#     - https://stackoverflow.com/questions/36399848/install-node-in-dockerfile
+
+FROM asdfghjklxd/reposense-images:ubuntu-22-04 AS setup-env
+
+LABEL os=ubuntu
+LABEL "os.version"="20.04"
```

This should probably be 22.04.
```suggestion
LABEL "os.version"="22.04"
```